bundle/edu.gemini.sp.vcs.reg/src/main/scala/edu/gemini/sp/vcs/reg/VcsRegistrar.scala

```diff
@@ -7,7 +7,7 @@ import edu.gemini.spModel.core.Peer
  * A service that provides persistent/mutable mapping between program ids and
  * VCS server locations.
  */
-trait VcsRegistrar {
+trait VcsRegistrar extends scala.collection.mutable.Publisher[VcsRegistrationEvent] {
```

Comment:
I have used the OSGi EventAdmin in the past but it is pretty archaic as OSGI APIs are often based on ancient java versions, I'm glad that there is this choice